pkg/csi_driver/multishare_ops_manager.go

```diff
@@ -251,7 +251,7 @@ func (m *MultishareOpsManager) startInstanceWorkflow(ctx context.Context, w *Wor
 func (m *MultishareOpsManager) verifyNoRunningInstanceOps(instance *file.MultishareInstance, ops []*OpInfo) error {
 	instanceUri, err := file.GenerateMultishareInstanceURI(instance)
```

Comment:
@saikat-royc Should all of the errors returned from GenerateMultishareInstanceURI be internal errors? Or can some be InvalidArgument?